src/qibocal/protocols/characterization/coherence/t1.py

```diff
@@ -91,35 +92,36 @@ def _acquisition(params: T1Parameters, platform: Platform, qubits: Qubits) -> T1
         params.delay_before_readout_step,
     )
 
+    sweeper = Sweeper(
+        Parameter.start,
+        ro_wait_range,
+        [ro_pulses[qubit] for qubit in qubits],
```

Comment:
```suggestion
        [ro_pulses[qubit] for qubit in qubits],
        type=SweeperType.ABSOLUTE
```
this more line does not change anything since absolute is already standard, but I think that helps avoiding confusion.

(I think this should be added also in the other sweepers)

Reply:
Ok, for clarity it may be helpful